backend/tests/Squidex.Domain.Apps.Entities.Tests/Contents/DomainObject/Guards/GuardContentTests.cs

```diff
@@ -98,10 +101,10 @@ public async Task CanCreate_should_not_throw_exception_if_data_is_not_null()
         [Fact]
         public async Task CanUpdate_should_throw_exception_if_data_is_null()
         {
-            SetupCanUpdate(true);
-
-            var content = CreateContent(Status.Draft);
-            var command = new UpdateContent();
+            SetupCanUpdate(true, user);
```

Comment:
Why are there so many changes and why have you changed the formatting? The CreateContent should provide a default that makes just sense.

Reply:
We need this params for method CheckPermission of GuardContent to return correct behavior.

Reply:
The actor for the command is fine, but what I mean is that the CreateContent command should have a default behavior, which is then overriden in the cases where we need it, e.g. something like

``
private IContentEntity CreateDraftContent(Status status, RefToken? createdBy = null)
        {
            return new ContentEntity
            {
                Id = DomainId.NewGuid(),
                NewStatus = status,
                AppId = appId
                AppId = appId,
                CreatedBy = createdBy ?? creator
            };
        }
  ```